packages/swarm-components/src/NumericalInput.jsx

```diff
@@ -2,47 +2,185 @@
 import * as React from 'react';
 import Icon from './Icon';
 
+type Value = number | null;
 type Props = {
     /**
-     * Classname of wrapper, specified because it is not applied with other
+     * indicates whether the input is disabled
```

Comment:
ah, looks like we should add this back. Technically it's not disallowed because this isn't a 'sealed' object type (unspecified properties are allowed), but we should definitely document it, potentially spreading `...other` on the wrapper div, and making the `Props` a union of HTMLElement props and the custom ones... @hsbacot I'd be happy to talk over this next week some time